kong/cache.lua

```diff
@@ -0,0 +1,164 @@
+local kong_mlcache = require "kong.mlcache"
+
+
+local type    = type
+local max     = math.max
+local ngx_log = ngx.log
+local ngx_now = ngx.now
+local ERR     = ngx.ERR
+local NOTICE  = ngx.NOTICE
+local DEBUG   = ngx.DEBUG
+
+
+local SHM_CACHE = "cache"
```

Comment:
Cool. Let's do that right after the merge, to have one single commit taking care of them all at once, to preserve atomicity then.